packages/playwright-core/src/server/registry/oopDownloadBrowserMain.ts

```diff
@@ -144,9 +144,9 @@ function toMegabytes(bytes: number) {
 
 async function main() {
   const log = (message: string) => process.send?.({ method: 'log', params: { message } });
-  const [title, browserDirectory, url, zipPath, executablePath, downloadConnectionTimeout] = process.argv.slice(2);
+  const [title, browserDirectory, url, zipPath, executablePath, downloadConnectionTimeout, downloadShowProgress] = process.argv.slice(2);
   await downloadFile(url, zipPath, {
-    progressCallback: getDownloadProgress(),
+    progressCallback: downloadShowProgress === 'true' ?  getDownloadProgress() : () => {},
```

Comment:
There already is a special case for `process.stdout.isTTY` (line 94 of this file). I think you are seeing the results of the `getBasicDownloadProgress` here, which is as intended. Sometimes downloading will fail and it is important to see when this happens.

Reply:
`isTTY` can still spam the console I think right? 

<img width="1213" alt="image" src="https://github.com/microsoft/playwright/assets/37447884/1e5e5112-065d-48f2-9fb8-f097589fbb73">


Current method shows an error if download happens even with `() => {}`

<img width="1149" alt="image" src="https://github.com/microsoft/playwright/assets/37447884/c6e20041-039a-4fd1-aa7e-645d21767e3a">


Anyway I might be misunderstanding the original goal so please let me know if you would like me to do this a different way.

Reply:
To make sure we are on the same page, we have two versions of the download progress reporting: `getBasicDownloadProgress` and animated `getAnimatedDownloadProgress`. The idea is that we use animated one whenever we detect TTY so that it was nice and interactive. And for the other cases (CI), we use the basic one.

I read you request as `getBasicDownloadProgress` is still too noisy, I want it to be more concise. Is that correct assessment? If so, the changes should be localized to `getBasicDownloadProgress`.